### DISCUSSION TITLE
Discord Replay/Score Embed

### DISCUSSION THREAD

**Discussion by RadioactivePeaches** [ppy/osu#30685] [Ideas] [answered]
I share my scores with friends on discord frequently, and I usually just take a screenshot of the end screen and sometimes I use the share replay feature on lazer. But when I share the actual replay link my friends have to actually click the link to see any details. It would be great if instead of a description of osu, score details are in the embed instead.

This way people you share to can see the basic information of the play such as 100s, misses, score, acc, etc and then if they want to download and watch the actual replay they can click the link. Or the embed could contain an image similar to how you screenshot the end screen or just the contents of the page itself, since its minimal anyway.

![image](https://github.com/user-attachments/assets/6e3a9d64-64be-4dea-8dfe-0008d84755d2)
This is what it looks like right now, but it would be cool if it looked something like what a lot of osu! bots provide instead:
![image](https://github.com/user-attachments/assets/c30385b3-467d-48d4-ab93-5c1bdcab50ed)

**Comment by cl8n** [comment]
given how popular discord is with osu players I think the premise here is fine. problem is that the embed is not very customizable coming from a website, at least not to the degree of something like Bathbot, all you can add is a few plain-text fields and an image

**Reply by RadioactivePeaches** [reply]
Could you display the data of the info as if it were a screenshot taken from the website? Idk how that kind of thing would work, maybe it's not worth the effort. A genshin stat website (enka.network) is able to load all your data in a way I think is similar to the score page for osu, which you can then download as a clean infographic. 
![image](https://github.com/user-attachments/assets/e1f76b00-90c4-4259-bd96-94d46a25607e)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu-web/issues/11678
